### PR TITLE
Python3 Runtime support

### PR DESCRIPTION
**Pull request by bleggett**
This is the feature to add support for targeting Python3 with Lambda functions in Jazz.

### CODE REVIEW

File: installscripts/dockerfiles/jenkins-ce/Dockerfile [hunk 1]
```diff
@@ -5,9 +5,13 @@ MAINTAINER JazzOSS Team
 # Switching to root to configure the image with system packages
 USER root
 RUN apt-get update && apt-get install -y vim curl sudo libtool autoconf make unzip rsync gcc autogen shtool pkg-config lsb-release python python-dev python-pip python-setuptools groff less && \
-curl https://bootstrap.pypa.io/get-pip.py | python && pip install --upgrade awscli && apt-get clean && pip install virtualenv && /usr/bin/easy_install virtualenv
```
Comment (svidhani) [inline installscripts/dockerfiles/jenkins-ce/Dockerfile]: what was the issue here?

Comment (devsatishm) [reply installscripts/dockerfiles/jenkins-ce/Dockerfile]: Looks like changes related to python2 & python3 are combined into a single line (below).

**Comment by bleggett** [comment]
@svidhani It's still labelled as `test-pending`, so until that label is removed by someone who tests it, I will assume no.

Just have this PR here for tracking, so we know where the effort is.

**Comment by devsatishm** [comment]
@bleggett @svidhani Changes are tested at a feature branch level. Merging into develop to perform release testing.

**Pull request merged**